src/pages/home/report/ReactionList/PopoverReactionList.js

```diff
@@ -81,23 +159,63 @@ class PopoverReactionList extends React.Component {
         });
     }
 
+    /**
+     * Get the selected reaction.
+     *
+     * @param {Array<Object>} reportActions
+     * @param {String} reportActionID
+     * @param {String} emojiName - Name of emoji
+     * @returns {Object}
+     */
+    getSelectedReaction(reportActions, reportActionID, emojiName) {
+        const reportAction = lodashFind(reportActions, (action) => action.reportActionID === reportActionID);
+        const reactions = lodashGet(reportAction, ['message', 0, 'reactions'], []);
+        const reactionsWithCount = lodashFilter(reactions, (reaction) => reaction.users.length > 0);
+        return lodashFind(reactionsWithCount, (reaction) => reaction.emoji === emojiName);
+    }
+
+    /**
+     * Get the reaction information.
+     *
+     * @param {Object} selectedReaction
+     * @returns {Object}
+     */
+    getReactionInformation(selectedReaction) {
+        if (!selectedReaction) {
+            return {
+                users: [],
+                emojiCodes: [],
+                emojiName: '',
+                emojiCount: 0,
+            };
+        }
+        const emojiCount = selectedReaction.users.length;
+        const reactionUsers = lodashMap(selectedReaction.users, (sender) => sender.accountID.toString());
+        const emoji = lodashFind(emojis, (e) => e.name === selectedReaction.emoji);
+        const emojiCodes = getUniqueEmojiCodes(emoji, selectedReaction.users);
+        const hasUserReacted = Report.hasAccountIDReacted(this.props.currentUserPersonalDetails.accountID, reactionUsers);
+        const users = PersonalDetailsUtils.getPersonalDetailsByIDs(reactionUsers);
```

Comment:
We missed to pass the accountID as the second param during this refractor causing this issue - https://github.com/Expensify/App/issues/20267